doc/creating-vm-from-scratch.md

```diff
@@ -5,6 +5,9 @@ for the EPICS Training from scratch.
 The recipe was set up using VirtualBox 7.0.14 on Windows 10,
 but it should work similarly on other versions and host systems.
 
+*This will not work for Silicon based Macs. Follow
+[these instructions](/doc/creating-vm-from-scratch-apple-silicon.md) for an altnernative.*
+
```

Comment:
alternative

Reply:
That's what I did. Install UTM, ...  as per the file in this pull request...

Reply:
creating-vm-from-scratch-**apple-silicon**.md

Reply:
The text should say "Apple Silicon" instead of just "Silicon" (the file name is fine).
The word "altnernative" contains a typo.